.github/workflows/main.yml

```diff
@@ -6,17 +6,36 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v1
-      - name: Build and push Docker images
-        uses: docker/build-push-action@v1.1.0
-        with:
-          username: ${{ github.actor }}
-          password: ${{ github.token }}
-          registry: docker.pkg.github.com
-          repository: tum-dev/campus-backend/backend-server
-          tags: latest
-          tag_with_ref: true
-          tag_with_sha: true
-          add_git_labels: true
-          build_args: version=${{ github.sha }}
\ No newline at end of file
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Docker meta
+      id: meta
+      uses: docker/metadata-action@v3
+      with:
+        images: tum-dev/campus-backend/backend-server
```

Comment:
This needs the fqdn to work (just like over at tum.sexy)

Reply:
Sorry. I fixed it.